new/LIT7072QeneMamhera.xml

```diff
@@ -0,0 +1,82 @@
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. In the first line, it 
+                    refers to <persName ref="PRS3934Ewostate"/>.
+                    It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD"/> in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-09-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7072QeneMamhera" passive="UppQ1524o"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 40r, 41r. Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .</note>
+                <ab>
+                    <l n="1">መምህረ፡ ዝንቱ፡ ዓለም፡ አውስጣቴዎስ፡ ደብረ፡ አርማንያ፡ ወልደከ፡ በጽሐ፡ ለዘምሮ፤</l>
+                    <l n="2">አምጣነ፡ ቦቱ፡ ተስእለ፡ ዘያስተፌስህ፡ አእመሮ፤</l>
+                    <l n="3">ወከመ፡ ዘንቃህ፡ እምንዋም፡ ተንስአ፡ ብኃይሉ፡ ወቀተለ፡ ጸሮ፤</l>
+                    <l n="4">ማዕረረ፡ ሞት፡ ኢያሪኮ፡ ቦቲኮ፡ ዓጽፍከ፡ 
+                        እግዚ<choice><orig>እ</orig><corr>አ</corr></choice>፡ 
```

Comment:
In this case, the እ was crossed out by the scribe and a ኣ (or አ) was written above. I think, this should be coded as:
 `<del rend="overstrike">እ</del><add place="above">ኣ</add>`.

The same with ብ / ቢ i.e.: `<del rend="overstrike">ብ</del><add place="above">ቢ</add>`

Otherwise it will look like you have corrected it.

Reply:
I used the encoding for "ancient correction" according to the guidelines, but I can change to what you suggest if that's better